src/wallet/wallet.h

```diff
@@ -73,6 +73,11 @@ static const unsigned int DEFAULT_TX_CONFIRM_TARGET = 6;
 static const bool DEFAULT_WALLET_RBF = false;
 static const bool DEFAULT_WALLETBROADCAST = true;
 static const bool DEFAULT_DISABLE_WALLET = false;
+//! -maxtxfee default
+constexpr CAmount DEFAULT_TRANSACTION_MAXFEE = COIN / 10;
```

Comment:
```suggestion
constexpr CAmount DEFAULT_TRANSACTION_MAXFEE{COIN / 10};
```

Could use c++11 to assert the result fits into CAmount without truncation?

Reply:
fixed